src/flutter/shell/platform/linux_embedded/surface/context_egl_drm_eglstream.cc

```diff
@@ -0,0 +1,83 @@
+// Copyright 2021 Sony Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux_embedded/surface/context_egl_drm_eglstream.h"
+
+#include "flutter/shell/platform/linux_embedded/logger.h"
+#include "flutter/shell/platform/linux_embedded/window/native_window_drm_eglstream.h"
+
+namespace flutter {
+
+ContextEglDrmEglstream::ContextEglDrmEglstream(
+    std::unique_ptr<EnvironmentEglDrmEglstream> environment)
+    : ContextEgl(std::move(environment), EGL_STREAM_BIT_KHR) {
+  if (!SetEglExtensionFunctionPointers()) {
```

Comment:
Please add the error handling before that like native_window_drm_gbm.cc

```
if (!valid_) {
  return;
}
```

Reply:
I fixed it.